backend/tests/test_email_handler.py

```
@@ -51,6 +52,21 @@ def test_email_data_with_optional_fields(self):
         assert email_data.bcc == ["bcc@example.com"]
 
 
+@pytest.fixture(autouse=True)
+def flask_app_context():
+    """Provide Flask app context so `current_app` LocalProxy is always bound."""
+    app = Flask(__name__)
+    app.config.update(
+        MAIL_SERVER="smtp.example.com",
+        MAIL_PORT=587,
+        MAIL_USERNAME="test@example.com",
+        MAIL_PASSWORD="test_password",
```

Ruff (0.15.1)

[error] Possible hardcoded password assigned to argument: "MAIL_PASSWORD"

(S106)